index.html

```diff
@@ -73,10 +73,69 @@ <h1 class="worktext">My Recent Works  </h1>
-        <button class="buttonw"><ins class="seep">See Project</ins></button>
+        <button class="buttonw" type="button"><ins class="seep">See Project</ins></button>
     </div>
 </div>
 <!--End Recent of works section-->
+<section class="projectcards">
+ <div class="row1projects">
+    <div class="card1">
+        <button class="card1button" type="button"><ins>See Project</ins></button>
+    </div>
+    <div class="card2">
+        <p class="card2h">Data Dashboard <br>Healthcare</p>
+        <p class="card2text">A daily selection of privately personalized
+             reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card2langs">
+            <li class="clanga"><a href="#">html</a></li>
+            <li class="clangb"><a href="#">bootstrap</a></li>
+            <li class="clangc"><a href="#">Ruby</a></li>
+        </ul>
+    </div>
+    <div class="card3">
+        <p class="card3h">Website Protfolio</p>
+        <p class="card3text">A daily selection of privately personalized
+             reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card2langs">
+            <li class="clanga"><a href="#">html</a></li>
+            <li class="clangb"><a href="#">bootstrap</a></li>
+            <li class="clangc"><a href="#">Ruby</a></li>
+        </ul>
+    </div>
+ </div>
+ <div class="row2projects">
+    <div class="card4">
+        <p class="card4h">Profesional Art Printing Data More</p>
+        <p class="card3text">A daily selection of privately personalized
+            reads; no accounts or sign-ups required. has been the industry's standard</p>
+       <ul class="card2langs">
+           <li class="clanga"><a href="#">html</a></li>
+           <li class="clangb"><a href="#">bootstrap</a></li>
+           <li class="clangc"><a href="#">Ruby</a></li>
+       </ul>
+    </div>
+    <div class="card5">
+        <p class="card2h">Data Dashboard <br>Healthcare</p>
+        <p class="card2text">A daily selection of privately personalized
+             reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card2langs">
+            <li class="clanga"><a href="#">html</a></li>
+            <li class="clangb"><a href="#">bootstrap</a></li>
+            <li class="clangc"><a href="#">Ruby</a></li>
+        </ul>
+    </div>
+    <div class="card6">
+        <p class="card3h">Website Protfolio</p>
+        <p class="card3text">A daily selection of privately personalized
+             reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card2langs">
+            <li class="clanga"><a href="#">html</a></li>
+            <li class="clangb"><a href="#">bootstrap</a></li>
+            <li class="clangc"><a href="#">Ruby</a></li>
+        </ul>
+    </div>
+ </div>
+</section>
       <div class="boutplus">
         <div class="abouttext">
         <div class="about">
```

Comment:
- [ ] [OPTIONAL] On mobile view kindly consider removing the bullets (style-list) from the form items, this will make your design reflect the Figma one and it will be consistent with the desktop version

![Screenshot_20220903_223210](https://user-images.githubusercontent.com/25227022/188288363-ff8d88af-a40b-4e77-9e65-84cc01236a43.png)